lora-phy/src/sx1276_7_8_9/mod.rs

```diff
@@ -23,6 +23,11 @@ const FREQUENCY_SYNTHESIZER_STEP: f64 = 61.03515625; // FXOSC (32 MHz) * 1000000
 const SX127X_MIN_LORA_SYMB_NUM_TIMEOUT: u16 = 4;
 const SX127X_MAX_LORA_SYMB_NUM_TIMEOUT: u16 = 1023;
 
+// Constant values need to compute the RSSI value
+const RSSI_OFFSET_LF: i16 = -164;
```

Comment:
Could you update the names of these constants to specify that these are for sx1276.
As this driver supports also sx1272, we'll need to fix RSSI/SNR calculation eventually for it as well.
(sx1272 supports only single range and it's RSSI_OFFSET value is -139).

Reply:
Thanks for the reviews.
Added a prefix to the constants.